Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -812,16 +812,14 @@ - (void)addUsersToProfileWhitelist:(NSArray<NSString *> *)recipientIds
             }];
         }
 
-        dispatch_async(dispatch_get_main_queue(), ^{
             for (NSString *recipientId in newRecipientIds) {
                 [[NSNotificationCenter defaultCenter]
-                    postNotificationName:kNSNotificationName_ProfileWhitelistDidChange
-                                  object:nil
-                                userInfo:@{
-                                    kNSNotificationKey_ProfileRecipientId : recipientId,
-                                }];
+                    postNotificationNameAsync:kNSNotificationName_ProfileWhitelistDidChange
```

Comment:
This was already being done async to the main thread (see above).  This is used by `ConversationViewController` to update its offer.  Should be fine.